src/utils/a11y.ts

```diff
@@ -95,3 +97,14 @@ export function focusPrevElem(elem, parentRef = null) {
 		}
 	}
 }
+
+export function setTabIndex(element: HTMLElement, index: -1 | 0 | 1) {
```

Comment:
Why limit to just those?
```suggestion
export function setTabIndex(element: HTMLElement, index: number) {
```

Reply:
That was my suggestion ... since tabindex `-1 | 0` are the only really useful options (we shouldn't be specifying positive indexes, since that will break DOM ordering). So, my bad on suggesting to include `1` in the range

Reply:
If this is in utils, our users can also use it in their apps (why wouldn't they) so it makes sense to allow them to set ordering. On a closer look, this function looks table specific so it should be a static function on table or sth.

Reply:
Is there a table utils it could live in? Having it as a static method on table means that other classes need to import table to use it, which doesn't look immediately correct

Reply:
Sure, that would work too. Using `Table.setTabIndex()` looks right, though, considering it's a table specific function for setting tab index. Either way makes sense, it's a stylistic preference, I'm good with either, they have the same spirit.